Tests/OnlineRetailPortal.Tests/ProductControllerFixture.cs

```diff
@@ -10,72 +11,115 @@ namespace OnlineRetailPortal.Tests
 {
     public class ProductControllerFixture
     {
-        static IProductStoreFactory storeFactory;
-        private IProductService _productService = new Services.ProductService(storeFactory);
+        private static IProductStoreFactory storeFactory = new ProductStoreFactory();
```

Comment:
if this is also a private variable can rename to _storeFactory

Reply:
done